Gruntfile.js

```diff
@@ -54,7 +54,13 @@ module.exports = function( grunt ) {
 		// Clean up the build.
 		clean: {
 			compiled: {
-				src: [ 'assets/js/*.js', '!assets/js/amp-service-worker-runtime-precaching.js', 'assets/js/*.asset.php' ],
+				src: [
+					'assets/js/**/*.js',
+					'assets/js/**/*.js.map',
+					'!assets/js/amp-service-worker-runtime-precaching.js',
+					'assets/js/**/*.asset.php',
+					'assets/css/*.css',
```

Comment:
Oh good. This should address an issue where running a build could accidentally include files that aren't on the branch, since they were leftover from builds on previous branches.